src/main/java/Duke.java

```diff
@@ -1,10 +1,362 @@
+                if(strT[1].trim() == "1") t.setDone(true);
+                addToList(t, Tasks, Tasks.size());
+                break;
+            case "D":
+                String[] strD = str.split(" \\| ");
+                for(String st: strD) System.out.println(st);
+                Deadline d = new Deadline(strD[2], strD[3]);
+                if(strD[1].trim() == "1") d.setDone(true);
+                addToList(d, Tasks, Tasks.size());
+                break;
+            case "E":
+                String[] strE = str.split(" \\| ");
+                for(String st: strE) System.out.println(st);
+                Event e = new Event(strE[2], strE[3]);
+                if(strE[1].trim() == "1") e.setDone(true);
+                addToList(e, Tasks, Tasks.size());
+                break;
+            }
+        }
+        return Tasks;
+    }
+    private static void printFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        System.out.println("Tasks recorded: ");
+        while (s.hasNext()) {
+            System.out.println(s.nextLine());
+        }
+    }
+
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    public static void checkNum(int num, int j) throws AssignException {
+        if(num > j || num <= 0)
+            throw new AssignException();
+    }
+
+    public static void checkList(int num) throws EmptyListException {
+        if(num < 1)
+            throw new EmptyListException();
+    }
+
+    public static void checkTodoString(String str) throws TodoStringException {
+        str = str.replaceAll(" ", "");
+        if(str.length() <= 4)
+            throw new TodoStringException();
+    }
+
+    public static void checkDeadlineString(String str) throws DeadlineStringException {
+        str = str.replaceAll(" ", "");
+        if(str.length() <= 8)
+            throw new DeadlineStringException();
+    }
+
+    public static void checkEventString(String str) throws EventStringException {
+        str = str.replaceAll(" ", "");
+        if(str.length() <= 5)
+            throw new EventStringException();
+    }
+
+    public static void checkFormat(String str) throws FormatException {
+        if(!str.contains("/by") && !str.contains("/at"))
+            throw new FormatException();
+    }
+
+    public static void FormatError() throws TypingException {
+        throw new TypingException();
+    }
+
+    public static void addToList(Task t, ArrayList<Task> list, int num){
+        list.add(t);
+    }
+
+    public static void printString(String str){
+        int length = str.length();
+        System.out.print("      ");
+        for(int i = 0; i < length; i++) System.out.print("_");
```

Comment:
The single line `for` statement is missing the curly brackets and line breaks required to meet the [coding standard](https://se-education.org/guides/conventions/java/basic.html#layout), which is at least 3 separate lines - they make code easier to read.